spec/support/vcr.rb

```diff
@@ -1,21 +1,21 @@
 VCR.configure do |c|
```

Comment:
I am having big trouble to figure out the proper VCR config to make VCR work on TRAVIS and LOCALLY. 

Previous I had that code 

```ruby
config.around(:each) do |example|
   if example.metadata[:type] == :request
     name = example.metadata[:full_description].split(/\s+/, 2).join('/').underscore.tr('.', '/').gsub(/[^\w\/]+/, '_').gsub(/\/$/, '')
     VCR.use_cassette(name, {}, &example)
   else
     VCR.turned_off(&example)
   end
 end
```

in the spec_helper file and all the tests were working LOCALLY but when it was hitting Travis, Travis would fail the VCR test because the request URL wouldn't be the same (why? I don't know). 

Then @DanielWright suggested to do it like we do it on Glossier. I gave it a go but now most of my VCR tests fail locally. It seems like if I re-do the same request twice in the same block, it will always retrieve the value of the first time instead of re-fetching it the second time. 

What are you guys suggestions? I have been at it for a long time now.

@braidn @DanielWright